release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -241,6 +241,18 @@ To evict pods instead of simulating the evictions, change the descheduler mode t
 
 For more information, see xref:../nodes/scheduling/nodes-descheduler.adoc#nodes-descheduler[Evicting pods using the descheduler].
 
+[id="ocp-4-11-nodes-descheduler-customizations"]
+==== New descheduler customizations
+
+This release introduces the following customizations for the descheduler:
+
+* Priority threshold filtering: Set the priority threshold either by class name (`thresholdPriorityClassName`) or by numeric value (`thresholdPriority`) to not evict pods that have a priority that is equal to or greater than that value.
+
+* Namespace filtering: Set a list of namespaces to include or exclude from descheduler operations.
```

Comment:
@bergerhoffer may be we should say user created namespaces so that some one whi is looking at this does not get confused with the list that already exists may be ? 
cc: @ingvagabund do you think the above make sense ?

Reply:
@kasturinarra Good point. 

@ingvagabund I did see the part about "Namespaces overrides included and excluded namespaces while keeping the default exclusion of all openshift-*, kube-system and hypershift namespaces". Does this mean those listed namespaces are always excluded, even if you were to try to add them to the included list?

Also, can you use a wildcard in the namespace field, or does it have to list each individual namespace one by one?

Reply:
> may be we should say user created namespaces

Agree.

> Does this mean those listed namespaces are always excluded, even if you were to try to add them to the included list?

Yes. All protected namespaces (all openshift-*, kube-system and hypershift namespaces) are always excluded. If a user tries to include any of the protected namespaces, the operator validation fails.

>  can you use a wildcard in the namespace field, or does it have to list each individual namespace one by one?

No wildcards. Always enumeration of all individual namespaces.

Reply:
Updated to add "user-created" and a note about the protected namespaces that are always excluded.

@kasturinarra Ready for re-review!

http://file.rdu.redhat.com/~ahoffer/2022/OSDOCS-3678-rn/release_notes/ocp-4-11-release-notes.html#ocp-4-11-nodes-descheduler-customizations